content/about/index.md

```diff
@@ -11,6 +11,5 @@ The objective of the OPI Project is to foster a community-driven
 standards-based open ecosystem for next generation architectures and frameworks
 based on DPU/IPU-like technologies.
 
-To achieve this goal, OPI is being formed by the merge of two projects:
-* [Diamond Bluff](https://diamond-bluff.github.io)
-* [IPDK](https://ipdk.io)
+Please see our main GitHub repository [README.md](https://github.com/opiproject/opi/blob/main/README.md)
+file for more information.
```

Comment:
maybe on the readme page here https://github.com/opiproject/opi/blob/main/README.md we should mention ipdk somehow... ?

Reply:
There is a mention of the IPDK.io where you can integrate in multiple open source projects such as IPDK.io, DPDK, SPDK, etc.  I think that the changes look good

Reply:
Yes, IPDK is mentioned in there, we should make the README.md in the main repository more explicit that IPDK is transitioning into the LF as a sub-project of OPI.